desktop/src/ui.rs

```diff
@@ -232,6 +232,12 @@ impl UiBackend for DesktopUiBackend {
         );
     }
 
+    fn display_root_movie_download_failed_message(&self) {
+        // Desktop doesn't download the root movie via core loader,
+        // so this should never be reached.
+        unreachable!();
```

Comment:
Are we *sure* this would never be reached? If this invariant is broken we probably wouldn't catch it in testing. We should at least pop up an actual dialog box here just-in-case. Even if it's just "That movie couldn't be loaded."

(For the record, I checked, and #4925 *shouldn't* break this invariant.)

Reply:
Sure, I'll add a dialog box and merge it then.